docs/send-data/hosted-collectors/amazon-aws/aws-s3-source.md

```diff
@@ -186,6 +186,12 @@ The following steps use the AWS SNS Console. You may instead use AWS CloudForma
 
 ### SNS with one bucket and multiple Sources
 
+    :::important
+    S3 Event Notifications do not allow you to have overlapping suffixes in two rules if the prefixes are overlapping for the same event type.
+
+    In this scenario, you will likely need to have a single Event Notification for the suffix that sends to an SNS Topic, and then have multiple SNS Subscriptions to the same Topic. This will allow you to have parallel data streams for the same Event Notifications (i.e. one that points to a Sumo Endpoint, and one that points elsewhere).
```

Comment:
```suggestion
    In this scenario, you will likely need to have a single Event Notification for the suffix that sends to an SNS Topic, and then have multiple SNS Subscriptions to the same Topic. This will allow you to have parallel data streams for the same Event Notifications (i.e., one that points to a Sumo Endpoint, and one that points elsewhere).
```